lib/webauthn/fake_client.rb

```diff
@@ -41,7 +42,8 @@ def create(
         client_data_hash: client_data_hash,
         user_present: user_present,
         user_verified: user_verified,
-        attested_credential_data: attested_credential_data
+        attested_credential_data: attested_credential_data,
+        extensions: extensions
```

Comment:
Nice! :clap:

This `extensions` argument is being returned both as the authenticator extension outputs and the client extension outputs in the response from what I understand. That won't happen in reality, right?

Should we allow two arguments here instead? `client_extension_outputs: nil` and `authenticator_extension_outputs: nil`, so on top of being more real we can also have more flexibility and control when testing different cases.

Reply:
>This extensions argument is being returned both as the authenticator extension outputs and the client extension outputs in the response from what I understand. That won't happen in reality, right?

Exactly! Yeah I agree that it's not likely to happen

>Should we allow two arguments here instead? client_extension_outputs: nil and authenticator_extension_outputs: nil, so on top of being more real we can also have more flexibility and control when testing different cases.

That was one of the approaches that I took, but I decided to not stay with it because that's not so real either, given that in reality, if I understood correctly, the relying party only requests a client extension to the client and it's that client who later requests an authenticator extension to the authenticator depending on that client extension requested. So in the end, I decided to go with the above approach. 
However, I agree that doing this approach will allow us to have more flexibility!

Reply:
> However, I agree that doing this approach will allow us to have more flexibility!

Yup.

I don't think I consider this a blocker though. We can improve later as needed.